src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -121,7 +121,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>{{'YOUR_INFORMAT
 
       </div>
 
-      <div class="row" ng-if="$ctrl.donorDetails['donor-type'] === 'Household'">
+      <div class="row" ng-if="$ctrl.donorDetails['donor-type'] === 'Household' && $ctrl.useV3 !== 'true'">
```

Comment:
What about the organization's name? What if a company donates and has its offices in an office park (In a building with lots of other offices)? They would likely need to add their organization name so they can receive post

Reply:
I think a case can be made here that certain campaigns will only be sent to individuals and use the new version which hides the organization data. It is possible that the stakeholders will want more granularity for these configs, but that should come out in UAT.